sdk/cognitiveservices/Language.TextAnalytics/src/Microsoft.Azure.CognitiveServices.Language.TextAnalytics.csproj

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <AssemblyTitle>Microsoft Azure Cognitive Services Language Text Analytics Client Library</AssemblyTitle>
     <Description>Provides API functions for consuming Microsoft Azure Cognitive Services Language Text Analytics APIs.</Description>
-    <Version>4.1.0-preview.1</Version>
+    <Version>4.1.0-preview.2</Version>
```

Comment:
This is more a clarifying question for me:
@AlexGhiondea when do we bump versions? after releasing and making a change to the code? before we release again/